### DISCUSSION TITLE
Dynamic Mail Config

### DISCUSSION THREAD

**Discussion by jajosheni** [laravel/framework#39612] [Q&A]
When setting the mail config via Config::set on running jobs I suppose one is changing the config in a project level.
When having multiple dynamic mail configs fetched from somewhere there is a chance of config overlapping:

Example: fetch config 1 , set config 1 , send email 1 with config 1.
                But if there is config 2 also being set at the same time you might risk sending email 1 with config 2.

**Comment by LocalHeroPro** [comment]
Bump.
Looks like, we have to use external repo for that. Especially on L9.

**Reply by jajosheni** [reply]
@LocalHeroPro yeah , i went with php mailer

**Reply by LocalHeroPro** [reply]
In L8, you can go with: https://laravel-news.com/allowing-users-to-send-email-with-their-own-smtp-settings-in-laravel.

**Reply by LocalHeroPro** [reply]
Yes. As I wrote:
> In L8
